odoo/tools/convert.py

```diff
@@ -647,14 +647,13 @@ def convert_csv_import(cr, module, fname, csvcontent, idref=None, mode='init',
         'install_filename': fname,
         'noupdate': noupdate,
     }
-    env = odoo.api.Environment(cr, SUPERUSER_ID, context)
-    result = env[model].load(fields, datas)
+    result = env[model].with_context(**context).load(fields, datas)
```

Comment:
It changes the behavior.
To keep the same as before:
```suggestion
    result = env[model].with_context(context).load(fields, datas)
```

Reply:
Indeed, but is it a bad thing ?
For instance if you want to import a file with some flags in the context activated ? e.g `mail_create_nolog=True` ?

Reply:
We can keep it like this.